oapth/src/fixed_sql_commands/pg.rs

```diff
@@ -51,9 +51,39 @@ where
       buffer.write_fmt(format_args!("DROP SEQUENCE {};", sequence))?;
   }
 
+  for view in views(back_end, "public").await? {
+      buffer.write_fmt(format_args!("DROP VIEW {};", view))?;
+  }
+
   Ok(buffer)
 }
 
+// https://github.com/flyway/flyway/blob/master/flyway-core/src/main/java/org/flywaydb/core/internal/database/postgresql/PostgreSQLSchema.java
+#[oapth_macros::dev_tools_]
+#[inline]
+pub async fn views<B>(back_end: &mut B, schema: & str) -> crate::Result<Vec<String>>
+where
+ B: crate::BackEnd
+{
+    let mut buffer = ArrayString::<[u8; 128]>::new();
```

Comment:
> Fmt: an error occurred when formatting an argument

This error is happening because the buffer is not large enough to accommodate the whole query string. A 256 bytes array should probably be good enough for this use-case

```suggestion
    let mut buffer = ArrayString::<[u8; 256]>::new();
```